docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+
+services:
+  standalone-chrome:
+    image: 'selenium/standalone-chrome:3.14.0-helium'
+    ports:
+      - '4444'
+    # Necessary due to some issues with chrome in docker.  See https://github.com/SeleniumHQ/docker-selenium#running-the-images
+    volumes:
+      - '/dev/shm:/dev/shm'
+    # Necessary to lock down screen sizes for our screen shot comparisons.
+    environment:
+      TZ: 'America/Chicago'
+
+  dev:
+    image: cerner/terra-node-dev:1
+    ports:
+      - 8080:8080
+    depends_on:
+      - standalone-chrome
+    volumes:
+      - .:/opt/module
+
+  standalone-chrome-travis:
```

Comment:
Since we need to test things in different browsers, Can we also add major browser like Firefox in here atleast?

Reply:
Yes we can, but we may as well wait to add it until we've got tests setup here. This is just the initial repo setup.